packages/mobile/locales/es-419/accountScreen10.json

```diff
@@ -7,17 +7,21 @@
   "sendIssueReport": "Reportar un incidente",
   "analytics": "Estadisticas de uso",
   "shareAnalytics": "Compartir estadisticas de uso",
-  "shareAnalytics_detail":
-    "Recopilamos datos anónimos sobre cómo utiliza Celo para ayudar a mejorar la aplicación para todos.",
+  "shareAnalytics_detail": "Recopilamos datos anónimos sobre cómo utiliza Celo para ayudar a mejorar la aplicación para todos.",
   "dataSaver": "Data Saver",
   "enableDataSaver": "Habilitar Data Saver",
-  "dataSaverDetail":
-    "El modo Data Saver te permite comunicarte con la Red Celo  a través de un nodo confiable. Puedes cambiar este modo en la configuración de la aplicación.",
-  "restartModal": {
+  "dataSaverDetail": "El modo Data Saver te permite comunicarte con la Red Celo  a través de un nodo confiable. Puedes cambiar este modo en la configuración de la aplicación.",
+  "restartModalSwitchOff": {
     "header": "Reiniciar para apagar Data Saver",
     "body": "Para encender y apagar Data Saver repetidamente, deberá reiniciar la aplicación.",
     "restart": "Reiniciar para alternar"
   },
+  "restartModalSwitchOn": {
+    "header": "~~Beware Of Restart",
```

Comment:
translations

Reply:
Yeah, were waiting on approve of the approach and now will do the translations :)